build_tools/cmake/build_and_test_asan.sh

```diff
@@ -125,6 +125,7 @@ label_exclude_regex="($(IFS="|" ; echo "${label_exclude_args[*]?}"))"
 declare -a excluded_tests=(
   "iree/samples/simple_embedding/simple_embedding_vulkan_test"
   "iree/tests/e2e/models/fragment_000.mlir.test"
+  "iree/tests/e2e/models/fully_connected.mlir.test"
```

Comment:
might want to turn off a few more ASan Vulkan tests in this directory :(
https://github.com/iree-org/iree/issues/5716
`e2e/models/mnist_fake_weights.mlir` has also failed recently

Not quite sure about the exclusion syntax, but we could just disable `iree/tests/e2e/models/*`